### DISCUSSION TITLE
Where are my saved Stockfish games?

### DISCUSSION THREAD

**Discussion by mnevin196** [official-stockfish/Stockfish#3796] [Q&A]
Please forgive the stupid nature of this question but I have entered my chess games into stockfish using my mobile phone running ios 15.0.2.  My question is - Where are my games stored?  How can I access the games stored area?  Is there a web site where this is explained?  I'd like to be able to search through my Stockfish database to be able to identify openings and opponents.  Thanks for any help anyone cam offer - Mike

**Comment by vondele** [comment]
what you are asking for is a property of the chess app you use (GUI), not of the underlying chess engine (Stockfish). We likely won't be able to answer the question.

**Comment by mnevin196** [comment]
Joost

Many thanks for your answer.  Can you tell me how I identify my chess app provider?

Thanks

Mike

Mike Nevin
Managing Director
Alliance Best Practice Ltd
Standing on the Shoulders of Giants
Mobile: +44 77 66 75 23 50
Office: +44 19 26 77 13 43
***@***.******@***.***>




On 17 Nov 2021, at 14:30, Joost VandeVondele ***@***.******@***.***>> wrote:



what you are asking for is a property of the chess app you use (GUI), not of the underlying chess engine (Stockfish). We likely won't be able to answer the question.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/official-stockfish/Stockfish/discussions/3796#discussioncomment-1658409>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWRIPMY6XH4IER5IBRUVUATUMO375ANCNFSM5IG2E2MQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by vondele** [comment]
Depends on the app you use. if you got this from the app store, go to the apps download page and search for 'App support' on that page.

**Comment by mnevin196** [comment]
Joost

Many thanks for your help

Mike

Mike Nevin
Managing Director
Alliance Best Practice Ltd
Standing on the Shoulders of Giants
Mobile: +44 77 66 75 23 50
Office: +44 19 26 77 13 43
***@***.******@***.***>




On 17 Nov 2021, at 16:11, Joost VandeVondele ***@***.******@***.***>> wrote:



Depends on the app you use. if you got this from the app store, go to the apps download page and search for 'App support' on that page.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/official-stockfish/Stockfish/discussions/3796#discussioncomment-1659116>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWRIPM4CSZQUGPYW64HV6KDUMPHZTANCNFSM5IG2E2MQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
